### ISSUE TITLE
Behavior on Android is different from specified in Readme

### ISSUE THREAD

**Issue by apripy** [larryaasen/upgrader#55]
According to the Readme:

> On Android, this widget does nothing as there is no easy way to query the Google Play Store for metadata about an app. Without the metadata, the widget cannot compare the app version with the latest Play Store version. It will not disrupt the widget tree and can be included in an Android without any issues.

However, in my application (not using an Appcast) the upgrader popup was shown to Android users, redirecting them to the AppStore. Looking at the code, it seems like the check has been commented:
```
//      // If this platform is not iOS, skip the iTunes lookup
//      if (!Platform.isIOS) {
//        return false;
//      }
```

**Comment by larryaasen** [comment]
Thanks for pointing out this issue. I was not able to reproduce the popup on Android because _appStoreVersion and _appStoreListingURL are null, which means isUpdateAvailable() will return false. However, the Android package name was 'com.larryaasen.upgraderexample' in the Android example app, and was not 'com.google.maps' like in the iOS example app. When the Android package name matched a valid bundle ID for iOS, this error would happen. I am working on a fix now.

**Comment by larryaasen** [comment]
I just published version 2.5.1 of this package with a fix for this issue. Please try it.
